src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
         gradeNames: ["fluid.viewComponent"],
         range: {}, // should be used to specify the min, max range e.g. {min: 0, max: 100}
-        selectors: {
-            thumb: ".ui-slider-handle"
-        },
         members: {
             combinedSliderOptions: {
                 expander: {
                     funcName: "fluid.slider.combineSliderOptions",
                     args: ["{that}.options.sliderOptions", "{that}.options.range"]
                 }
-            },
-            slider: {
-                expander: {
-                    "this": "{that}.container",
-                    method: "slider",
-                    args: ["{that}.combinedSliderOptions"]
-                }
+            }
+        },
+        modelRelay: {
+            target: "value",
```

Comment:
This is rather awkward - especially with respect to @kaspermarkus upcoming fix which will make this transform invertible. If you change the type of the field, you should lens it to a different path.